anta/cli/nrfu/commands.py

```diff
@@ -63,6 +60,28 @@ def text(ctx: click.Context) -> None:
     exit_with_code(ctx)
 
 
+@click.command()
+@click.pass_context
+@click.option(
+    "--csv-output",
```

Comment:
Should we make this more generic like `-o or --output` like the JSON option? This is also what I did in my MD PR but open to change it.

Reply:
I am just thinking it might be untrivial once we move to multiple commands approach having the same knob for different commands